### PR TITLE
Add 'textStays' option to speech zone

### PR DESCRIPTION
**Pull request by smeas**

### CODE REVIEW

File: Assets/Scripts/UI/Speech/SpeechZone.cs [hunk 1]
```diff
@@ -181,10 +191,12 @@ public class SpeechZone : MonoBehaviour {
 	private IEnumerator CoShowMessages() {
 		messageText.text = "";
 
+		bool didShowAnyMessage = false;
```
Comment (MrJackey) [inline Assets/Scripts/UI/Speech/SpeechZone.cs]: What does this bool do? Can't seem find any usage.

File: Assets/Scripts/UI/Speech/SpeechZone.cs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Collections;
+using System.Linq;
```
Comment (MrJackey) [inline Assets/Scripts/UI/Speech/SpeechZone.cs]: Are you sure you are using this? Don't see any usage.

**Review by MrJackey — CHANGES_REQUESTED** [review]
Just two tiny things, the rest looks good 👍

File: Assets/Scripts/UI/Speech/SpeechZone.cs (hunk 1 above)
Comment (smeas) [reply Assets/Scripts/UI/Speech/SpeechZone.cs]: Oh damn, forgot to remove that.

**Review by MrJackey — APPROVED** [review]
+1

**Review by Addehman — APPROVED** [review]
Clever!

**Pull request merged**